plugins/frontend/backstage-plugin-prometheus/src/hooks/usePrometheus.test.ts

```diff
@@ -81,5 +84,15 @@ describe('usePrometheus', () => {
       const { displayableAlerts } = useAlerts(['not existing alert']);
       expect(displayableAlerts).toHaveLength(0);
     });
+
+    it('should filter out alerts without requested label', () => {
+      (useAsync as any).mockReturnValue({
+        value: require('../mocks/mockAlertResponse.json'),
+        loading: false,
+      });
+
+      const { displayableAlerts } = useAlerts('all');
+      expect(displayableAlerts).toHaveLength(1);
+    });
```

Comment:
Please include a test case for scanario when there's no label selector defined.

Reply:
Resolved